layer4/test-data/multi-tool-plan.yaml

```diff
@@ -0,0 +1,214 @@
+metadata:
+  id: test-multi-tool-osps-plan
+  version: "1.0.0"
+  author:
+    name: Test Evaluation Plan Generator
+    uri: https://example.com/test-plans
+    version: "1.0.0"
+  mapping-references:
+    - id: OSPS-B
+      title: Open Source Project Security Baseline
+      version: "2025"
+      description: The Open Source Project Security (OSPS) Baseline
+
+plan:
+  plans:
+    - control:
+        reference-id: OSPS-B
+        entry-id: OSPS-AC-03
+      assessments:
+        - requirement:
+            reference-id: OSPS-B
+            entry-id: OSPS-AC-03.01
+          strategy:
+            conflict-rule-type: Strict
+            remarks: |
+              Branch protection is critical for security. Any failure from any procedure
+              (automated or manual) should cause the requirement to fail, ensuring high security standards.
```

Comment:
This feels like information that would go in a Policy document, neh? 

I could see remarks still being useful, though. Probably for notes related to how the assessment is expected to be executed.